### DISCUSSION TITLE
[Feature] Queuing new items while slideshow is already playing.

### DISCUSSION THREAD

**Discussion by mnjm97** [immich-app/immich#9017] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

would it be possible to have a running slideshow queue new items that are added to an album.

this would allow, for instance, a grandparent to have a picture slideshow running and for family members to post new pictures to a shared album and have the running slideshow just keep adding new pictures. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by chem-william** [comment]
I second this request! It would allow Immich to be used as a digital frame

**Comment by radh21301** [comment]
I suggest you look into immich-frame. It is for this exact purpose. I run it on an old android tablet, and it is amazing.

**Reply by chem-william** [reply]
It's close, but it also doesn't seem to auto update the shown picures if you add new ones to the album. Or maybe I'm doing something wrong?

**Reply by radh21301** [reply]
I think it does. It caches some photos, but not all. It talks to Immich via api and pulls some random images. And you can restrict it to pull only from certain albums or images of certain people. But I dont think you can force it to pull the recent images. It is random.

Could also be something with your setup. Maybe ask on the forum: https://discord.com/channels/979116623879368755/1217843270244372480

**Reply by chem-william** [reply]
Hmm.. I've played with it some more and it won't auto-update albums if you add new pictures to it. The only way I could get new pictures to show up is if the app is pointing to the user. Then it'll pull new images as soon as they're uploaded

**Reply by radh21301** [reply]
I see. Probably something to ask the dev on the discord channel I shared earlier. They are very responsive and would definitely help out.
